requirements/requirements.insights.txt

```diff
@@ -90,60 +95,57 @@ diff-match-patch==20200713
     # via django-import-export
 distro==1.7.0
     # via bindep
-django==3.2.13
-    # via
-    #   django-automated-logging
-    #   django-currentuser
-    #   django-filter
-    #   django-guardian
-    #   django-guid
-    #   django-import-export
-    #   django-lifecycle
-    #   django-picklefield
-    #   django-storages
-    #   djangorestframework
-    #   drf-nested-routers
-    #   drf-spectacular
-    #   pulpcore
 django-automated-logging==6.1.3
     # via galaxy-ng (setup.py)
 django-currentuser==0.5.3
     # via pulpcore
 django-filter==21.1
     # via pulpcore
-django-guardian==2.4.0
-    # via pulpcore
-django-guid==3.2.2
+django-guid==3.3.0
     # via pulpcore
 django-import-export==2.8.0
     # via pulpcore
 django-ipware==3.0.7
     # via django-automated-logging
-django-lifecycle==0.9.6
+django-lifecycle==1.0.0
     # via pulpcore
 django-picklefield==3.0.1
     # via django-automated-logging
 django-prometheus==2.2.0
     # via galaxy-ng (setup.py)
 django-storages[boto3]==1.12.3
     # via -r requirements/requirements.insights.in
-djangorestframework==3.13.1
+django==3.2.13
```

Comment:
Can you confirm you are using the latest pip-tools? I see has reordered where some reqs go. I assume if we update to latest then this will be last reorder.